src/abstractdataframe/io.jl

```diff
@@ -211,3 +192,140 @@ end
 function Base.show(io::IO, ::MIME"text/tab-separated-values", df::AbstractDataFrame)
     printtable(io, df, true, '\t')
 end
+
+##############################################################################
```

Comment:
Feel free to leave this commented. I'm working on merging all my updates for DataStreams and I'll put the right code here.